File: appserver-legacy/src/main/java/org/radarbase/appserver/exception/entity/ErrorDetails.java
package org.radarbase.appserver.exception.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class ErrorDetails {
    Instant timestamp;
    int status;
    String message;
    String path;
}

File: src/main/kotlin/org/radarbase/appserver/exception/entity/ErrorDetails.kt
package org.radarbase.appserver.exception.entity

import org.radarbase.appserver.util.OpenClass
import java.time.Instant

@OpenClass
class ErrorDetails(
    var timestamp: Instant? = null,
    var status: Int = 0,
    var message: String? = null,
    var path: String? = null
)